apps/portals/b2ai.standards/src/config/GrandChallengeResources.ts

```diff
@@ -0,0 +1,54 @@
+import B2AI_ORG_114_svg from '@/assets/B2AI_ORG_114.svg?url'
+import B2AI_ORG_115_svg from '@/assets/B2AI_ORG_115.svg?url'
+import B2AI_ORG_116_svg from '@/assets/B2AI_ORG_116.svg?url'
+import B2AI_ORG_117_svg from '@/assets/B2AI_ORG_117.svg?url'
+
+import B2AI_ORG_114_html from '@/assets/d4d/B2AI_ORG_114_d4d.html?raw'
+import B2AI_ORG_116_html from '@/assets/d4d/B2AI_ORG_116_d4d.html?raw'
+import B2AI_ORG_117_html from '@/assets/d4d/B2AI_ORG_117_d4d.html?raw'
+
+import B2AI_ORG_114_css from '@/assets/d4d/B2AI_ORG_114_d4d.css?raw'
+import B2AI_ORG_116_css from '@/assets/d4d/B2AI_ORG_116_d4d.css?raw'
+import B2AI_ORG_117_css from '@/assets/d4d/B2AI_ORG_117_d4d.css?raw'
+
+export const imageUrls = {
+  'B2AI_ORG:114': B2AI_ORG_114_svg,
+  'B2AI_ORG:115': B2AI_ORG_115_svg,
+  'B2AI_ORG:116': B2AI_ORG_116_svg,
+  'B2AI_ORG:117': B2AI_ORG_117_svg,
+}
+
+export const GCInfo = {
```

Comment:
Is this list of organizations with datasheet information expected to remain small? Is it expected to change often?

For those organizations with this embedded content, is the information in the HTML page expected to change? It's not clear where these HTML files came from and I don't know how I would update them.

Reply:
About where the files came from, you can look at [this Slack thread](https://b2aiworkspace.slack.com/archives/C0738H2KVGB/p1752187350029169) and [B2AI issue 244](https://github.com/bridge2ai/b2ai-standards-registry/issues/244). 

The HTML is expected change (and so is the css). 

Beyond the four GCs, we don't expect to see more D4D content, but the GCs are expected to have additional D4Ds (I think just version changes?), but not very many.

Reply:
Can you add a comment to this GrandChallengeResources file outlining the steps for updating this content? These threads are a lot of discussion for an engineer to wade through. If it is an engineer's responsibility to update this content (which is atypical for our portals where content updates are usually done through Synapse tables), it should be as simple of a process as possible.

Reply:
As discussed in slack, let's do a table that includes the organization ID. The HTML (as well as other assets, like CSS if needed) can go into either a `LARGETEXT` or a filehandle

Reply:
I created a separate issue for this: [Make D4D content table on Synapse and scripts/gh-action to load it from data-sheets-schema #368](https://github.com/bridge2ai/b2ai-standards-registry/issues/368)?